docs/pvacvector.rst

```diff
@@ -10,10 +10,21 @@ DNA vector based personalized cancer vaccines. It takes as input either a pVACse
 tsv file or a fasta file containing peptide sequences and returns a peptide ordering that
 minimizes the effects of junctional epitopes (that may create novel peptides)
 between the sequences. It does this by using the core pVACseq services to
-predict the binding scores for each junctional peptide. It also tests
-junctions with spacer amino acid sequences that may help to eliminate junctional epitopes.
-These spacer amino acid sequences can be "HH", "HHC", "HHH", "HHHD", "HHHC",
-"AAY", "HHHH", "HHAA", "HHL" or "AAL. The final vaccine ordering is
+predict the binding scores for each junctional peptide separated by a spacer amino acid
+sequences that may help to eliminate junctional epitopes. The list of spacers
```

Comment:
I'd go with either `by a spacer amino acid sequence` or `by spacer amino acid sequences`.
```suggestion
sequence that may help to eliminate junctional epitopes. The list of spacers
```

Reply:
I went with the first suggestion just because there is only one spacer used at a time.